lib/functions/autoload.php

```diff
@@ -19,8 +19,14 @@
  * @return void
  */
 function load_nonadmin_files() {
-	$filenames = require CHILD_THEME_DIR . '/config/autoload-nonadmin-files.php';
+	$config = get_stylesheet_directory() . '/config/autoload-nonadmin-files.php';
 
+	if ( ! is_readable( $config ) ) {
+		\ChristophHerr\Prometheus2\config_unavailable_message();
```

Comment:
If a file isn't available, I'd say that's a issue for the developer to fix - well before a user gets to run it.

Throw an exception (from a named constructor), rather than letting it trickle up to the user interface.

Reply:
Some of the exceptions will be visible on the front end as well as the back end, so that might get the attention of everybody involved ;)
See #22